src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-form/multi-cluster-form.component.html

```diff
@@ -0,0 +1,150 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container i18n="form title"
+                class="modal-title">Connect Cluster
+  </ng-container>
+  <ng-container class="modal-content">
+    <form name="remoteClusterForm"
+          #frm="ngForm"
+          [formGroup]="remoteClusterForm">
+      <div class="modal-body">
+        <cd-alert-panel *ngIf="connectionMessage !== 'Connection Verified Successfully' && connectionMessage !== ''"
```

Comment:
```suggestion
        <cd-alert-panel *ngIf="connectionVerified"
```

Reply:
Why using connection message to check rather than bool?

Reply:
Updated the PR